packages/roosterjs-editor-plugins/lib/plugins/TableCellSelection/keyUtils/handleKeyUpEvent.ts

```diff
@@ -26,7 +26,7 @@ export function handleKeyUpEvent(
         !state.preventKeyUp &&
         IGNORE_KEY_UP_KEYS.indexOf(which) == -1
     ) {
-        clearState(state, editor);
+        editor.addUndoSnapshot(() => clearState(state, editor));
```

Comment:
Using Arrow Keys, Escape or some other keys that do not add content will add a unneeded undo snapshot. Is Undo snapshot really needed?

Reply:
Makes sense. In that case, my change in handleKeydownEvent.ts to addUndoSnapshot() is not needed

Reply:
I can use isCharacterValue here to only add the snapshot if it is true.